src/config.ts

```diff
@@ -76,12 +77,24 @@ export class Config {
       INCLUDE_PROJECT_IDS: includeProjectIds,
       INCLUDE_DATASOURCE_IDS: includeDatasourceIds,
       INCLUDE_WORKBOOK_IDS: includeWorkbookIds,
+      TABLEAU_SERVER_VERSION_CHECK_INTERVAL_IN_HOURS: tableauServerVersionCheckIntervalInHours,
     } = cleansedVars;
 
     const defaultPort = 3927;
     const httpPort = cleansedVars[httpPortEnvVarName?.trim() || 'PORT'] || defaultPort.toString();
     const httpPortNumber = parseInt(httpPort, 10);
 
+    const defaultTableauServerVersionCheckIntervalInHours = 1;
+    const tableauServerVersionCheckIntervalInHoursNumber = tableauServerVersionCheckIntervalInHours
+      ? parseFloat(tableauServerVersionCheckIntervalInHours)
+      : defaultTableauServerVersionCheckIntervalInHours;
+
+    this.tableauServerVersionCheckIntervalInHours =
+      isNaN(tableauServerVersionCheckIntervalInHoursNumber) ||
+      tableauServerVersionCheckIntervalInHoursNumber <= 0
+        ? defaultTableauServerVersionCheckIntervalInHours
+        : tableauServerVersionCheckIntervalInHoursNumber;
```

Comment:
This will get replaced with the `parseNumber` helper function from the OAuth branch. Ref: https://github.com/tableau/tableau-mcp/pull/142/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R162